ARCs/arc-0003.md

```diff
@@ -78,6 +91,10 @@ The JSON Metadata schema is as follows:
             "type": "string",
             "description": "Describes the asset to which this token represents"
         },
+        "extra_metadata": {
+            "type": "string",
+            "description": "Extra metadata in base64. If the property is specified (even if it is an empty string) the asset metadata (am) of the ASA is computed differently than if it is not specified."
```

Comment:
I'm not sure we gain anything by specifying that extra-metadata is b64 encoded.  Why not leave it unspecified and concatenate that utf-8 encoding of the field, rather than decoding first.  I don't think that prevents people from doing what you specified, if they choose.  But if their metadata is not binary, I'd rather see it in the file unencoded.

Reply:
Then we would need to have two fields: one base64, one not. I think it would become quite complex